.github/workflows/dispatch-build-bottle.yml

```
@@ -0,0 +1,112 @@
+name: Dispatch build bottle
+
+on:
+  workflow_dispatch:
+    inputs:
+      macos:
+        description: macOS version
+        required: true
+      formula:
+        description: Formula name
+        required: true
+      issue:
+        description: Issue number, where comment on failure would be posted
+        required: false
+      upload:
+        description: Whether to upload built bottles to Bintray or not
```

```suggestion
      publish:
        description: Whether to publish uploaded bottles to Bintray automatically
```

if I've understood this correctly (as it looks like the upload would be unconditional). Please correct me if I'm wrong 😀

---

I chose the "upload" word, cause we have `brew pr-upload` command and it performs the upload/publish to Bintray. I would leave say we use those words interchangeably, for example in `publish-commit-bottles.yml` the job is named "upload", sooo yea.

---

@dawidd6 I suggested this because `Upload bottles` and `Upload bottles to Bintray` steps below are a bit confusing, could one be renamed? `Upload bottles to GitHub Actions` and `Upload and publish bottles on Bintray` perhaps?

---

Done. I also found I didn't pass the formula name to test-bot.

https://github.com/Homebrew/homebrew-core/commit/d4204bef963b82cfd05345f8992fbed6a0b5c494